### PR TITLE
GETOOL 311: Website audit

### PR DESCRIPTION
**Pull request by Mitso**

### CODE REVIEW

**Comment by Mitso** [comment]
@sewagodimo @moh-moola @codiebeulaine 
Please review Profiles and Comments tweaks.

File: gem/templates/rwanda/registration/login.html [hunk 1]
```diff
@@ -1,45 +1,45 @@
...
-    <input type="hidden" name="next" value="{% if request.GET.next %}{{ request.GET.next }}{% else %}{% url "molo.profiles:login_success" %}{% endif %}" class="call-to-action__button call-to-action__button--sprimary" />
+  </fieldset>
+  <input type="submit" value="{% trans "Log in" %}" class="call-to-action__button call-to-action__button--primary">
+  <input type="hidden" name="next" value="{% if request.GET.next %}{{ request.GET.next }}{% else %}{% url "molo.profiles:login_success" %}{% endif %}" class="call-to-action__button call-to-action__button--sprimary" />
   </form>
-    <a href="{% url "molo.profiles:forgot_password" %}" class="call-to-action__nav-item-text call-to-action__nav-item-text--left">
-      {% trans "Forgotten your pin?" %}
-    </a>
+  <a href="{% url "molo.profiles:forgot_password" %}" class="call-to-action__nav-item-text call-to-action__nav-item-text--left">
+    {% trans "Forgotten your pin?" %}
```
Comment (sewagodimo) [inline gem/templates/rwanda/registration/login.html]: Isn't the convention "Forgot my pin"

File: gem/templates/rwanda/profiles/reset_password_success.html [hunk 2]
```diff
@@ -0,0 +1,16 @@
+{% extends "base.html" %}
+{% load i18n %}
+
+{% block content %}
+<div class="profiles">
+  <div class="heading">
+    <h1 class="heading__profiles">{% trans "PIN Changed" %}</h1>
+  </div>
+  <div class="heading">
+    <h3 class="heading__subheading">{% trans "Your PIN has been changed sucessfully." %}</h3>
```
Comment (sewagodimo) [inline gem/templates/rwanda/profiles/reset_password_success.html]: Is there a difference between PIN and password?

Comment (Mitso) [reply gem/templates/rwanda/profiles/reset_password_success.html]: @sewagodimo yes there is, the convention is to use pin throughout the client facing content.

File: gem/templates/rwanda/registration/login.html (hunk 1 above)
Comment (Mitso) [reply gem/templates/rwanda/registration/login.html]: Well spotted, thank you.

**Pull request merged**